libs/stac/odc/stac/_load.py

```diff
@@ -0,0 +1,100 @@
+"""
+stac.load - dc.load from STAC Items
+"""
+from functools import partial
+from typing import Any, Callable, Dict, Iterable, Optional, Sequence, Union
+import numpy as np
+
+import pystac
+import xarray as xr
+from datacube.model import Dataset
+from datacube.utils.geometry import GeoBox
+from odc.index import patch_urls
+
+from ._dcload import dc_load
+from ._eo3 import ConversionConfig, stac2ds
+
+# pylint: disable=too-many-arguments
+def load(
+    items: Iterable[pystac.Item],
+    bands: Optional[Union[str, Sequence[str]]] = None,
+    geobox: Optional[GeoBox] = None,
+    groupby: Optional[str] = None,
+    resampling: Optional[Union[str, Dict[str, str]]] = None,
+    skip_broken_datasets: bool = False,
+    chunks: Optional[Dict[str, int]] = None,
+    progress_cbk: Optional[Callable[[int, int], Any]] = None,
+    fuse_func: Optional[Callable[[np.ndarray, np.ndarray], np.ndarray]] = None,
+    stac_cfg: Optional[ConversionConfig] = None,
+    patch_url: Optional[Callable[[str], str]] = None,
+    **kw,
+) -> xr.Dataset:
+    """
+    Load STAC Items (from the same or similar collections) as an
+    xarray Dataset.
+
+    :param items:
+       STAC Items to load
+
+    :param bands:
+        List of band names to load, defaults to All. Also accepts
+        single band name as input.
+
+    :param geobox:
+       Allows to specify exact region/resolution/projection
+
+    :param groupby:
+       Controls what items get placed in to the same pixel plane,
+       supported values are "time" or "solar_day", default is "time"
+
+    :param resampling:
+       Controls resampling strategy, can be specified per band.
+
+    :param skip_broken_datasets:
+       Continue processing when IO errors are encountered
+
+    :param chunks:
+       Rather than loading pixel data directly construct
+       a Dask backed arrays. ``chunks={'x': 2048, 'y': 2048}``
```

Comment:
- find lines that need changing
- select them
- press `+/-` button or `Ctrl-g`
- change them

you get this:
```suggestion
       Rather than loading pixel data directly construct
       Dask backed arrays. ``chunks={'x': 2048, 'y': 2048}``
```

it is then easy to incorporate the change directly from github.com

Reply:
@RichardScottOZ like this?

Reply:
Yeah, wasn't somewhere that was easy to do at the time sorry, but yes.

construct... arrays.
or construct a Dask backed array.

Whichever you prefer/wanted in this case